server/src/main/java/org/elasticsearch/search/SearchService.java

```diff
@@ -219,6 +219,13 @@ public class SearchService extends AbstractLifecycleComponent implements IndexEv
         Property.Dynamic
     );
 
+    public static final Setting<Boolean> QUERY_PHASE_PARALLELISM_ENABLED = Setting.boolSetting(
+        "search.query_phase_parallelism_enabled",
+        false, // TODO: ES-6596 will set this to true once the feature has had time to "bake".
```

Comment:
this is actually the right default, we don't want to enable query phase concurrency by default just yet. But we do want to be able to enable it.